akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/EventSourcedSettings.scala

```diff
@@ -76,7 +79,8 @@ private[akka] final case class EventSourcedSettings(
     logOnStashing: Boolean,
     recoveryEventTimeout: FiniteDuration,
     journalPluginId: String,
-    snapshotPluginId: String) {
+    snapshotPluginId: String,
+    useContextLoggerForInternalLogging: Boolean = true) {
```

Comment:
So, the default in the `.conf` file is `false`, but here it is `true`. Is that intentional?

Reply:
No, that's a mistake. Code should not have a hardcode value.